contracts/engine/V3/forks/PROHIBITION/GenArt721CoreV3_Engine_Flex_PROHIBITION.sol

```diff
@@ -0,0 +1,2276 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.17;
+
+// Created By: Art Blocks Inc.
+
+import "./interfaces/IAdminACLV0_PROHIBITION.sol";
+import "./interfaces/IGenArt721CoreContractV3_Engine_Flex_PROHIBITION.sol";
+
+import "../../../../interfaces/0.8.x/IRandomizerV2.sol";
+import "../../../../interfaces/0.8.x/IEngineRegistryV0.sol";
+import "../../../../interfaces/0.8.x/IDependencyRegistryCompatibleV0.sol";
+import "../../../../interfaces/0.8.x/IManifold.sol";
+
+import "@openzeppelin-4.7/contracts/access/Ownable.sol";
+import "../../../../libs/0.8.x/ERC721_PackedHashSeed.sol";
+import "../../../../libs/0.8.x/BytecodeStorage.sol";
+import "../../../../libs/0.8.x/Bytes32Strings.sol";
+
+/**
+ * @title Art Blocks Engine Flex ERC-721 core contract, V3.
```

Comment:
Nit: A general suggestion to add a `V3_PROHIBTION_CHANGELOG.md` file to this `engine/V3/forks/PROHIBITION` directory that summarizes the small diffs here for future reference, while it is fresh-of-mind.

Reply:
Done.

Reply:
🙏